ReactNativeClient/lib/components/shared/note-screen-shared.js

```diff
@@ -195,9 +195,13 @@ shared.isModified = function(comp) {
 };
 
 shared.initState = async function(comp) {
-	const isProvisionalNote = comp.props.provisionalNoteIds.includes(comp.props.noteId);
+	const isProvisionalNote = !comp.props.noteId || comp.props.provisionalNoteIds.includes(comp.props.noteId);
+
+	let note = await Note.load(comp.props.noteId);
+	if (!note) {
+		note = comp.state.note;
+	}
 
-	const note = await Note.load(comp.props.noteId);
```

Comment:
The fix should not be here, as a note with an undefined ID is invalid. Instead, the note should be created like so by the quick action:

    const newNote = await Note.save({ ...... }, { provisional: true });

Reply:
Thanks, I'll update